src/cloudai/util/docker_image_cache_manager.py

```diff
@@ -251,9 +251,22 @@ def cache_docker_image(
 
         try:
             p = subprocess.run(enroot_import_cmd, shell=True, check=True, capture_output=True)
+            stdout = p.stdout.decode() if p.stdout else ""
+            stderr = p.stderr.decode() if p.stderr else ""
```

Comment:
You can do `subprocess.run(..., text=True)` to avoid manually calling `decode()`.